articles/how-to-use-array-operators-in-js/index.md

```diff
@@ -0,0 +1,507 @@
+#### prerequisites
+- Have [Node.js](https://nodejs.org/en/) installed in your computer.
+- Have some basic knowledge of JavaScript
+
+### What we will cover:
+- [Introduction into JavaScript arrays](#introduction-into-javascript-arrays).
+- [Properties of JavaScript arrays](#properties-of-javascript-arrays).
+- [Operations on JavaScript arrays](#operations-on-javascript-arrays).
+
+
+### Introduction into JavaScript arrays.
+Arrays are list-like objects for storing elements in a program. Since JavaScript is not statically typed, an array can hold different types of data. For instance, an array can have integers, strings, and floats at the same time. The number of elements stored in an array is not fixed; it's upon the programmer's precedence.
+
+### Properties of JavaScript arrays.
+- **Length**: It indicates the number of elements in a particular array. Consider the following array:
+
+```javascript
+const fruits = ["mangoes","lemons","avocados"];
+```
+
+If we want to know the number of fruits stored in the fruits array we have to access the length property.
+
+```javascript
+console.log(fruits.length);
+```
+Output
+
+```bash
+    3
+```
+
+Since there are three elements in the fruits array, the output shall be three.
+
+- **Constructor**: It indicates the function that created the array object. Using our previous array of fruits, we can check the constructor property.
+
+```javascript
+console.log(fruits.constructor);    
+```
+Output
+
+```bash
+[Function: Array]
+```
+
+`[Function: Array]` is the default constructor value when creating arrays in javascript.
+
+- **Prototype**: It allows us to add our own custom operations and properties to the arrays. For example, considering our fruits array, we can opt to implement a function to capitalize each element's first letter. Since there is no such pre-built operation in JavaScript, we can use `prototype`.
+
+```javascript
+Array.prototype.toUCase = function() {
+  for (i = 0; i < this.length; i++) {
+    this[i] = this[i][0].toUpperCase() + this[i].slice(1);
+  }
+};
+
+//we execute the operation on our array
+fruits.toUcase();
```

Comment:
This results in the following error `TypeError: fruits.toUcase is not a function`
check on this

Reply:
solved.